packages/metadata-booster/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "metadata-booster",
+  "version": "0.4.3",
```

Should we change the package name to match other booster packages? Should the package version match Booster's?

```suggestion
  "name": "@boostercloud/metadata-booster",
  "version": "0.29.6",
```